_posts/philosophers/2020-07-15-heraclitus.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Heraclitus
+date: 2020-07-15 04:00
+revised: 2020-07-15 04:00
+category: Philosophers
+tags: [pre-socratic, greek, cosmologist]
+---
+Born some time around 535 BC.
+Died some time around 475 BC (age ~60).
+
+Heraclitus was an Ionian philospher. He was born into wealth and a hereditary political office, but eschewed it to pursue philosophy. He was well known for his depressive, misanthropic character and witty word play.
+
+Heraclitus was the first recorded Greek philosopher to declare that the world is in a state of flux: an ever-changing state of becoming. This idea is captured by his famous quote, "No man ever steps into the same river twice."
+
+Heraclitus also believed in conflict between opposites. However, whereas Anaximander saw conflict between opposites as a destabilizing force, Heraclitus saw it as a healthy state of tension. Heraclitus was the first recorded Greek philsopher to propose the idea of "unity in diversity". Multiple opposing forces balance the world, like the tension in a bow or a lyre. For this reason, Heraclitus declared that the world is ultimately composed of fire, since it best exemplifies this tension: fire needs fuel to burn, but if it burns all of the fuel then it dies. He postulated that the different kinds of observable matter must be the result of different kinds of fires, which balance each other. Thus, while matter may temporarily change, the overall amount of matter is preserved.
```

Comment:
It's interesting that the idea of conservation of matter has been around so long, and is apparently so intuitive for us. I wonder why? Also -- this gave me an idea for a series: the genealogy of an idea. Take some basic idea (like conservation of matter), trace its roots, and follow its development throughout history. I think that would be really interesting.